app/event.tsx

```diff
@@ -32,63 +53,146 @@ export default function ModalScreen() {
 		}
 	};
 
+	const makeReservation = async (event: Event, placeId: number) => {
```

Comment:
Wouldn't it be better to extract this function? It seems to be repeated

Reply:
No